LightAPRS-hab/LightAPRS-hab.ino

```diff
@@ -28,7 +29,31 @@
 #define AprsPinInput  pinMode(12,INPUT);pinMode(13,INPUT);pinMode(14,INPUT);pinMode(15,INPUT)
 #define AprsPinOutput pinMode(12,OUTPUT);pinMode(13,OUTPUT);pinMode(14,OUTPUT);pinMode(15,OUTPUT)
 
-//#define DEVMODE // Development mode. Uncomment to enable for debugging.
+#define DEVMODE // Development mode. Uncomment to enable for debugging.
```

Comment:
I think we want to commit code to master that does not enable the debugging.  we want to prepare it like it is going to be headed to space.

Reply:
Done.